packages/lodestar/src/util/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./numpy.js";
```

Comment:
@claravanstaden this is very simple utility, can you just copy the source in our consumer code?